understanding/labels-or-instructions.html

```diff
@@ -33,11 +33,12 @@ <h1>達成基準 3.3.2: ラベル又は説明を理解する</h1>
       <main>
          <section id="intent">
             <h2>意図</h2>
-            <p>この達成基準の目的は、期待される入力データが何なのかを利用者が知ることができるように、コンテンツ作成者にフォーム内のコントロールを特定する説明又はラベルを提示させることである。特に一般的なフォーマットではない場合、又は正しい入力のための特定のルールがある場合、入力欄のデータフォーマットを説明文又はラベルで明示してもよい。特に説明が長く詳細である場合、個々のコントロールがフォーカスを受け取ったときだけそのような説明文を利用者が利用できるように作ることをコンテンツ制作者は選択してもよい。</p>
+            <p>この達成基準の目的は、期待される入力データが何なのかを利用者が知ることができるように、コンテンツ作成者にフォーム内のコントロールを特定する説明又はラベルを提示させることである。ラジオボタン、チェックボックス、コンボボックス、又は類似のコントロールで利用者に選択肢を提供する場合は、それぞれの選択肢に適切なラベルがあって、利用者が実際に何を選択しているかをわかるようにしなければならない。なお、特に一般的なフォーマットではない場合又は正しい入力のための特定のルールがある場合は、説明文又はラベルで、データ入力欄のデータフォーマットを明示してもよい。特に説明が長く詳細である場合は、個々のコントロールがフォーカスを受け取ったときだけそのような説明文を利用者が利用できるように作ることをコンテンツ制作者は選択してもよい。</p>
             <p>この達成基準の意図は、ページを不要な情報だらけにしてしまうことではなく、障害のある利用者に役立つ重要な手がかり及び説明文を提供することである。情報又は指示が多すぎると、少なすぎる場合と同じくらい有害になることもある。目標は、利用者の混乱を最小限に抑えて、必要以上のナビゲーションを提供することなく、利用者がタスクを完了するために十分な情報を提供することである。</p>
             <p>この達成基準では、ラベル又は説明が正確にマークアップされる、識別される、又はそれぞれのコントロールに関連付けられることを要求しているわけではない。この観点は、<a href="info-and-relationships.html">1.3.1：情報及び関係性</a>で個別に扱われる。なお、コンテンツが達成基準 1.3.1 に失敗している (ラベル又は説明が正しくマークアップ、識別、又は関連付けられていない) 場合でも、この達成基準には合格している (関連するラベル及び説明が提供されている) 場合がある。</p>
             <p>さらに、この達成基準では、フォームのコントロール及び入力欄にアクセシブルな名前 (accessible name) 又は説明を提供するための代替方法が使用されているかどうかは考慮されない。この観点は、<a href="name-role-value.html">4.1.2: 名前 (name)・役割 (role)・値 (value)</a> で個別に扱われる。コントロール及び入力欄が適切なアクセシブルな名前 (accessible name) 又は説明 (例えば、<code>aria-label = &quot;...&quot;</code>を使用する) を持ち、よって達成基準 4.1.2 に合格することが可能であるが、この達成基準には失敗する (支援技術を使用しているユーザーだけでなく、すべてのユーザーにラベル又は指示が提示されていない) 場合がある。</p>
-            <p>この達成基準では、コントロールと入力欄にラベルをつける必要があるが、これらのラベルが十分に明確で説明的であるかどうかは、<a href="headings-and-labels.html">2.4.6：見出し及びラベルを理解する</a>で個別に扱われる。</p>
+			<p>この達成基準は、データ入力に関連しないリンク又はその他のコントロール (展開／折りたたみウィジェットあるいは類似のインタラクティブなコンポーネントなど) には適用されない。</p>
```

Comment:
「～など」があるとき、「又は」「もしくは」を使わないようにします
```suggestion
			<p>この達成基準は、データ入力に関連しないリンク又はその他のコントロール (展開／折りたたみウィジェット、類似のインタラクティブなコンポーネントなど) には適用されない。</p>
```